omero/users/cli/import.txt

```diff
@@ -162,4 +208,12 @@ arguments to run. These options will also be displayed on the command line by pa
 
 .. literalinclude:: /downloads/cli/help.out
 
-.. seealso:: :doc:`/sysadmins/import-scenarios`
+.. seealso:: 
```

Comment:
When I say 'if we are leaving these at the very bottom', I mean the alternative would be to put the seealso box at the bottom of the intro paragraph